UPGRADE.rst

```diff
@@ -0,0 +1,47 @@
+Upgrade notes
+=============
+
+The upgrade notes only describe necessary changes that you might need to make
+to your setup in order to use a new role release. Refer to the Changelog for
+more details about what has changed.
+
+From debops.php5 role to debops.php role
+----------------------------------------
+
+- The ``debops.php5`` role stored the PHP-FPM pool configuration in a custom
+  ``/etc/php5/fpm/pool-available.d/`` directory and symlinked the files to the
+  ``/etc/php5/fpm/pool.d/`` directory used by PHP-FPM daemon. In ``debops.php``
+  this mechanism was dropped and files are generated directly in
+  ``/etc/php5/fpm/pool.d/``. Existing files seem to be unaffected by being
+  symlinked to the old directory therefore you don't need to replace the
+  symlinks with real files manually.
+
+- The ``debops.php`` role has new role dependencies on ``debops.php/env`` and
+  ``debops.logrotate``. Refer to the provided playbook for an example usage.
+
+- The different ``/etc/php5/*/php.ini`` configuration files managed by
+  ``debops.php5`` role are not managed by ``debops.php`` role anymore and might
+  be incomplete. You can replace them with the pristine versions located in:
+  ``/usr/share/php5/php.ini-*`` or ``/usr/lib/php/7.0/php.ini-*``.
+
+- All of the role variables have been renamed from ``php5_*`` to ``php__*``.
+  Some variables have been further renamed, for example from ``php5_default_*``
+  to ``php__fpm_*`` or various variables to ``php__ini_*``. You should check
+  the variables used in inventory against the new default variables and update
+  accordingly.
+
```

Comment:
Yeah sorry, I meant those which have changed more than just the 2 underscores (php_ to php__). Because I'd do a search & replace for that, but further renaming will require editing in each case.